src/environs/__init__.py

```diff
@@ -569,6 +569,27 @@ def _expand_vars(self, parsed_key, value):
     def _get_key(self, key: _StrType, *, omit_prefix: _BoolType = False) -> _StrType:
         return self._prefix + key if self._prefix and not omit_prefix else key
 
+    def _get_value(self, env_key: _StrType, default: typing.Any) -> typing.Any:
+        return os.environ.get(env_key, default)
+
+
+class FileAwareEnv(Env):
+    """An environment variable reader that supports reading values from files."""
+
+    def __init__(self, *, file_suffix: _StrType = "_FILE", **kwargs):
+        self.file_suffix = file_suffix
+        super().__init__(**kwargs)
```

Comment:
even though it's duplicative, i'd prefer to explicitly declare the inherited keyword arguments, so that static analysis/autocomplete etc work.

```suggestion
    def __init__(
        self, 
        *, 
        file_suffix: _StrType = "_FILE", 
        eager: _BoolType = True,
        expand_vars: _BoolType = False,
        prefix: _StrType | None = None,
    :
        self.file_suffix = file_suffix
        super().__init__(eager=eager, expand_vars=expand_vars, prefix=prefix)
```